app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
```

Comment:
Properties should be ordered border, margin-bottom, width<br>Each selector in a comma sequence should be on its own line